app/services/add_sandbox_template.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require_dependency "#{Rails.root}/lib/wiki_api"
+require_dependency "#{Rails.root}/lib/wiki_edits"
+
+class AddSandboxTemplate
+  def initialize(home_wiki:, sandbox:, sandbox_template:, current_user:)
+    @current_user = current_user
+    @sandbox = sandbox
+    @default_template = '{{user sandbox}}'
```

Comment:
This can be a constant rather than an instance variable.

Reply:
Done!